definitions/npm/@apollo/react-common_v3.x.x/flow_v0.104.x-/react-common_v3.x.x.js

```diff
@@ -74,14 +74,14 @@ declare module '@apollo/react-common' {
   /* end context */
 
   /* start parser */
-  declare export type DocumentType = 'Query' | 'Mutation' | 'Subscription'
+  declare export type DocumentType = 'Query' | 'Mutation' | 'Subscription';
 
   declare export type IDocumentDefinition = {
     type: DocumentType,
     name: string,
     variables: $ReadonlyArray<VariableDefinitionNode>,
```

Comment:
Would be helpful to fix this too
```suggestion
    variables: $ReadOnlyArray <VariableDefinitionNode>,
```

Reply:
It's weirding me out that Flow didn't catch this in the type defs in my project...